proto/account.proto

```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
 
 package types;
-//option go_package = "github.com/aergoio/aergo/types";
+option go_package = "/types";
```

Comment:
Why this change? Was it tested with this change?

Reply:
yes. previous code only worked only for Aergo repositories. ( In fact, it didn't work even in aergo )
[aergo repository](https://github.com/aergoio/aergo/tree/topic/update-go-dependency) also apply this change too. ( make protoc / make protoclean )

Reply:
https://github.com/aergoio/aergo/blob/0ce27749e9d09998aedc5bc35654961711dc89a0/CMakeLists.txt#L56